### PR TITLE
[Bugfix]enable `--no-open` option

### PR DESCRIPTION
**Pull request by hayato-osh**
Issue #5

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -173,26 +173,13 @@ export async function activate(
...
-                : (await exists("yarn.lock"))
-                ? "yarn storybook --no-open"
-                : "npm run storybook --no-open";
+            const config = vscode.workspace.getConfiguration(
+              "storybook-opener.storybookOption"
+            );
+            const port = config.get<number>("port");
+            const startCommand = config.get<string>("startCommand");
+
+            const command = startCommand || `npx storybook dev -p ${port} --no-open`;
```
Comment (ygkn) [inline src/extension.ts:182]: `-p ${port}` Nice. Can you add the other options (`host`, `https`)?

Comment (hayato-osh) [reply src/extension.ts]: @ygkn 
Thanks for your review.
When localhost is entered in the `host` option, the storybook did not open, so it is set to an empty string.

[5e6be55](https://github.com/ygkn/storybook-opener/pull/6/commits/5e6be55e8d06181fad87b1eb5d83d409879be8b6)

Comment (ygkn) [reply src/extension.ts]: cool!

**Review by ygkn — APPROVED** [review]
Thanks a lot for your contribution!

LGTM

**Pull request merged**